Misha/Text/OB-197_guestbook/guestbook_main.py

```diff
@@ -0,0 +1,69 @@
+import tkinter as tk
+import time
+
+
+class Main(tk.Frame):  # создание главного окна
```

Comment:
You can use mocks and patches to test this class. It would be good example to practice it! 
https://docs.python.org/3.3/library/unittest.mock-examples.html

Reply:
Also one valuable link for tkinter with mocks 
https://code.activestate.com/recipes/578964-robust-testing-of-tkinter-menu-items-with-mocking/